container/stream/streams.go

```diff
@@ -119,28 +119,31 @@ func (c *Config) CopyToPipe(iop *cio.DirectIO) {
 	ctx := context.TODO()
 
 	c.dio = iop
-	copyFunc := func(w io.Writer, r io.ReadCloser) {
+	copyFunc := func(name string, w io.Writer, r io.ReadCloser) {
 		c.wg.Add(1)
 		go func() {
 			if _, err := pools.Copy(w, r); err != nil {
-				log.G(ctx).Errorf("stream copy error: %v", err)
+				log.G(ctx).Errorf("stream copy %s error: %v", name, err)
 			}
 			r.Close()
 			c.wg.Done()
 		}()
 	}
 
 	if iop.Stdout != nil {
-		copyFunc(c.Stdout(), iop.Stdout)
+		copyFunc("stdout", c.Stdout(), iop.Stdout)
 	}
 	if iop.Stderr != nil {
-		copyFunc(c.Stderr(), iop.Stderr)
+		copyFunc("stderr", c.Stderr(), iop.Stderr)
 	}
 
 	if stdin := c.Stdin(); stdin != nil {
 		if iop.Stdin != nil {
 			go func() {
-				pools.Copy(iop.Stdin, stdin)
+				_, err := pools.Copy(iop.Stdin, stdin)
+				if err != nil {
+					log.G(ctx).Errorf("stream copy stdin error: %v", err)
+				}
 				if err := iop.Stdin.Close(); err != nil {
 					log.G(ctx).Warnf("failed to close stdin: %v", err)
```

Comment:
(if we decide to update - message itself also open for suggestions, like the other ones);

```go
log.G(ctx).WithFields(log.Fields{"error": err, "stream": "stdin"}).Warn("closing stream")
```